js/index.js

```diff
@@ -1,7 +1,44 @@
 // Iteration 1: Names and Input
 
+let hacker1 = ("Jan");
+console.log(`The driver's name is ${hacker1}`);
+
+let hacker2 = ("Bert");
+console.log(`The navigator's name is ${hacker2}`);
 
 // Iteration 2: Conditionals
 
+if (hacker1.length == hacker2.length) {
```

Comment:
Please be mindful about the difference between strict and "loose" comparison. In this case "==" works fine. However, by default, I would stick to "===". https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons?page=2&tab=Votes